src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlEventSource.cs

```diff
@@ -51,7 +51,7 @@ private SqlEventSource()
         // the EventSource activity IDs (because they currently don't use tasks and this simply confuses the logic) and 
         // because of versioning requirements we don't have ActivityOptions capability (because mscorlib and System.Data version 
         // at different rates)  Sigh...
-        [Event(SqlEventSource.EndExecuteEventId, Keywords = Keywords.SqlClient, Task = Tasks.ExecuteCommand, Opcode = EventOpcode.Stop)]
+        [Event(SqlEventSource.BeginExecuteEventId, Keywords = Keywords.SqlClient, Task = Tasks.ExecuteCommand, Opcode = EventOpcode.Start)]
         public void BeginExecute(int objectId, string dataSource, string database, string commandText)
```

Comment:
surprised no unit tests caught this?

Reply:
I think the test suite is more inclined towards NetCore, this might have been missed.